### PR TITLE
readme fixes

### PR DESCRIPTION
**Pull request by xpqz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,26 @@
...
+## Beta software
 
-Note: this is a proof of concept; it's not battle tested or supported in any way. If you find bugs (of which there will be plenty), do let us know – or better, consider a pull request.
+Note: this is beta; it's not battle tested or supported in any way. If you find bugs (of which there will be plenty), do let us know – or better, consider a pull request. You know what beta means, right?
 
 ## Installation
 
 ### Deploy to Bluemix
 
+**Note**: Envoy relies on some cutting edge features from CouchDB2. A Cloudant account attained through Bluemix will by default not be compatible. You will need to request that the account be moved to the Cloudant cluster "Porter" with an email to `support@cloudant.com` stating your account name.
```
Comment (willholley) [inline README.md]: suggest `Envoy relies on some CouchDB features which are not yet in an official release.`

File: README.md [hunk 2]
```diff
@@ -1,28 +1,26 @@
...
+
 The fastest way to deploy *Cloudant Envoy* to Bluemix is to click the **Deploy to Bluemix** button below.
```
Comment (willholley) [inline README.md]: is it? won't this fail because this binds to a Bluemix Cloudant instance?

File: README.md [hunk 3]
```diff
@@ -1,28 +1,26 @@
...
 
-**Note**: Envoy relies on some cutting edge features from CouchDB2. A Cloudant account attained through Bluemix will by default not be compatible. You will need to request that the account be moved to the Cloudant cluster "Porter" with an email to `support@cloudant.com` stating your account name.
-
 ### Manual installation
 
-Cloudant Envoy is a Node.js application on top of the Express.js framework. To install, clone the repo and run `npm install`. The Envoy server needs admin credentials for the backing Cloudant database, and it expects the following environment variables to be set:
+Cloudant Envoy is a Node.js application on top of the Express.js framework. To install from source, clone the repo and run `npm install`. The Envoy server needs admin credentials for the backing Cloudant database, and it expects the following environment variables to be set:
```
Comment (willholley) [inline README.md]: do we want to mention `npm install envoy`?

File: README.md [hunk 4]
```diff
@@ -105,42 +103,25 @@ DEBUG=cloudant,nano node app.js
 
 ## Introduction
 
-Cloudant has the potential to be an ideal backend for a mobile application. It is scalable, it syncs, and being schema-free it can cope with the frequent data changes that tend to happen in mobile development.
+Cloudant is scalable, syncs, and schema-free. It can cope with the frequent data changes that typically happens in mobile and web development. It stores data as JSON documents
 
-However, Cloudant was never designed to be an _mbaas_ – a complete mobile application backend, and comparisons with dedicated mobile application backends such as Facebook's (now defunct) _Parse_ stack highlight our shortcomings in this area. Some of the problematic areas include:
+However, Cloudant was never designed to be an _mbaas_ – a complete mobile application backend, and comparisons with dedicated mobile application backends such as Facebook's (now defunct) _Parse_ stack highlight the shortcomings in this area. Some of the problematic areas include:
```
Comment (willholley) [inline README.md]: I'm not sure we want to position envoy as an MBaaS layer on top of Cloudant - really, we're just solving sub-database access control.

**Comment by tomblench** [comment]
@xpqz whilst you're looking at this, can you change the build badge link to point to the travis status page?

**Pull request merged**